frontend/src/components/utilities/parseJson.ts

```diff
@@ -0,0 +1,11 @@
+export const parseJson = (src: ArrayBuffer | string) => {
+  const file = src.toString();
+  const formatedData: Record<string, string> = JSON.parse(file);
+  const env: Record<string, { value: string; comments: string[] }> = {};
+  Object.keys(formatedData).forEach((key) => {
+    if (typeof formatedData[key] === "string") {
+      env[key] = { value: formatedData[key], comments: [] };
```

Comment:
Any reason we don't support comments? We could add support for it by allowing the following input:

```json
{
   "KEY": "value" // We already support this
}
```

And

```json
{
   "KEY": {
      "value": "the value",
      "comment": "some comment"
   }
}
```

This would make the .yml / .json functionality more uniform.

Reply:
json itself actually doesn't support comments; it's data only - I didn't no this for awhile actually. Javascript objects deceive you into thinking it can haha

Reply:
2nd option is viable but I'd say lets wait to see if someone requests that?

Reply:
> json itself actually doesn't support comments; it's data only - I didn't know this for awhile, Javascript objects deceive you into thinking it can haha

Yeah it was just for illustration I added that


> 2nd option is viable but I'd say lets wait to see if someone requests that?
Sure, we can do that. Works for now